packages/workbox-background-sync/demo/sw.js

```diff
@@ -1,35 +1,36 @@
 /* global workbox*/
 importScripts(
-	'../build/background-sync-queue.js',
-	'../../workbox-routing/build/workbox-routing.js',
-	'../../workbox-runtime-caching/build/workbox-runtime-caching.js'
+  '../build/importScripts/workbox-background-sync.dev.v0.0.1.js',
```

Comment:
You should be able to load this in via `/__test/bundle/<pkg>` if you assume that these demos will always be run when served by our test server.